server/tpm2.go

```diff
@@ -147,6 +148,9 @@ func TPM2_GetRandom(size uint16) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	if rbytes == nil || len(rbytes) != int(size) || onlyContainsZeros(rbytes) {
+		return nil, errors.Errorf("Failed to generate random bytes")
```

Comment:
Why not errors.New() like you did in another file?

Reply:
New is more adapted here. I'll change this.